Keithley_2700Sup/devKeithley_2700.proto

```diff
@@ -0,0 +1,174 @@
+##################################################
+#
+# Protocol file for Keithley 2700
+#
+##################################################
+
+Terminator = '\r\n';
+ReplyTimeout = 2000;
+
+getIDN {
+    out "*IDN?";
+    #Read no more that 39 chars (EPICS limit)
+    in "%/(.{0,39})/";
```

Comment:
I wouldn't specify the character limit here. Either read it in as a string and ignore the extra input, or you can read it in as a character waveform (e.g. ILM). Is the ID likely to overflow the 39 character limit?